lib/components/dashboard/degree-info/CourseBar.tsx

```diff
@@ -116,7 +119,7 @@ const CourseBar: FC<{
         />
         <div className="truncate">{section}</div>
         <div>
-          {remainingCredits === 0 && completed ? (
+          {remainingCredits === 0 && distribution.satisfied ? (
```

Comment:
now use info from backend instead of calculating planned or satisfied in frontend